### PR TITLE
Update - shorter workpackages version and partners' contacts

### PR DESCRIPTION
**Pull request by tgrzinic-x41**
Please, check if the content is now ok.

### CODE REVIEW

File: workpackages.md [hunk 1]
```diff
@@ -13,85 +13,69 @@ PeT-HMR is divided in six work packages (WPs).
...
-are to be supported in the best possible way so that the manual work components
-can be minimized.
+Work package 3 focuses on the automated implementation of basic
+penetration tests. This component of the project is of particular interest
+to small and medium-sized enterprises, because it will offer a simpler and more
+accessible penetration test process that tries to minimize the involvement
+of penetration test experts.
 
 A clear goal of the work package is to redefine the limits of what is possible
 by machine. This is done, among other things, with self-learning components
```
Comment (onetwo) [inline workpackages.md]: by machine?

File: workpackages.md [hunk 2]
```diff
@@ -13,85 +13,69 @@ PeT-HMR is divided in six work packages (WPs).
...
+taking into account the rapidly changing state of the art, PeT-HMR will design
+AI approaches for detection of novel attacks and will use the collected attacks
+to perform security testing scenarios.
+These requirements are used to identify detailed criteria for evaluating the
+quality of the solutions found through experiments that simulate real-world
+attack scenarios. After the problems behind APeT and SHoP have been formalized
+and algorithms for solving them have been designed, they will be implemented
+respectively in Work packages 2 and 3.
 
 **WP Lead:** This work package is led by IFIS.
```
Comment (onetwo) [inline workpackages.md]: Remove Lead and Result please :-)

File: workpackages.md [hunk 3]
```diff
@@ -108,81 +92,60 @@ DSGVO will be taken.
...
+generation. Based on the results of APeT and SHoP, a classification is performed
+to mitigate vulnerabilities with respect to their severity in the context of 
+a system to be protected.
 
-Work plan: Patches and mitigations (technical or organizational measures to
-prevent exploitation of a vulnerability) can be generated using data from SHoP
-and APeT or selected in the simple case.
+Patches and mitigations are technical or organizational measures to prevent 
+exploitation of a vulnerability that can be generated using data from SHoP
+and APeT or manually selected.
```
Comment (onetwo) [inline workpackages.md]: ... or that can be manually selected

File: workpackages.md [hunk 4]
```diff
@@ -202,25 +165,19 @@ also be tested and evaluated for their applicability in commercially used
...
-environments before being tested in realistic environments and corporate
-networks of varying sizes and configurations. A demonstrator will be derived
-from these scenarios. The goal is to build the demonstrator in stages so that
-it initially contains initial functionalities and proofs-of-concept after WP
-6.1 and fully tests and represents the developed technologies at the end of WP
-6.3.
+cost-effectively in cloud infrastructures.
 
-**WP Lead:** This work package is led by NetUSE AG.
+The goal is to build the demonstrator in stages, so that the initial demonstrator
```
Comment (onetwo) [inline workpackages.md]: This part could be removed

**Comment by tgrzinic-x41** [comment]
@onetwo fixed. please check and proceed with the deployment if it is all ok. In the contact.md you will find some visual fixes that I made after checking the test build.

**Pull request merged**